config.yaml

```diff
@@ -0,0 +1,150 @@
+
```

Comment:
I could not find a case for this in the PR, but stratified config warrants a check for when the config is stored. If it's ever flattened there could be parameters override (two same keys under different sub dicts). Seems though well managed